letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -319,6 +319,32 @@ else
   BOOTSTRAP_VERSION="Unknown 1"
 fi
 
+PrevBootstrapVersion() {
+  if [ -f $VENV_BOOTSTRAP_VERSION ]; then
+    cat $VENV_BOOTSTRAP_VERSION
+  elif grep -Fqx "$BOOTSTRAP_VERSION" << "UNLIKELY_EOF"
+NO_BOOTSTRAP 1
+BootstrapDebCommon 1
```

Comment:
> I was actually going to suggest giving all the existing bootstrap scripts version 0, since it's more "special" to human eyes and people will probably give version 1 to new scripts no matter what our documentation says.

I personally have a preference of hard coding a list like this. Up until now, we've had no reason to change our bootstrap scripts and I'm only planning on adding a new one. With this in mind, most will continue to sit at whatever version we give them now for the foreseeable future. Rather than hoping/documenting that future devs should do versioning a specific way, I'd rather put this list here so bootstrap script authors don't need to know anything about what version strings are treated specially. Regardless of what they put, it should just work.

> That, combined with --no-bootstrap genuinely sidestepping the bootstrapping logic, is my favored approach.

Great I'll do this along with preserving the bootstrap version between venvs if `--no-bootstrap` is provided.